### ISSUE TITLE
SDAnimatedImageView播放apng动画时，在iPhone 13 pro机型上太快，应该是因为手机是120Hz刷新频率导致的，希望修复

### ISSUE THREAD

**Issue by zifeizhang** [SDWebImage/SDWebImage#3579]
### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by dreampiggy** [comment]
Provide your issue template. At least



Info | Value
-- | --
Platform Version | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
-- | --
SDWebImage Version | e.g. 5.0.0 / 4.4.0

**Comment by zifeizhang** [comment]
<img width="1172" alt="image" src="https://github.com/SDWebImage/SDWebImage/assets/13514514/b6590a71-f12d-411c-ba4d-32d57f36ec74">

**Comment by zifeizhang** [comment]
👆上面代码执行动画，设置时长没用，看了源码属性playbackRate 这个设置会有用；
1、在60Hz的手机上，使用默认playbackRate 这个值，动画正常播放；
2、在120Hz的手机上，速度感觉快了一倍的样子；
3、看了你的 SDDisplayLink 这个类，里面有 #define kSDDisplayLinkInterval 1.0 / 60 这个宏定义，猜测是否和这个有关

**Comment by zifeizhang** [comment]
iOS 16系统，苹果13pro

**Comment by dreampiggy** [comment]
Do you try to ugprade and test result again ?

> SDWebImage Version | e.g. 5.0.0 / 4.4.0

**Comment by lyandy** [comment]
@zifeizhang  I guess your issue is the same as https://github.com/SDWebImage/SDWebImage/issues/3605

ref: [Optimizing ProMotion refresh rates for iPhone 13 Pro and iPad Pro](https://developer.apple.com/documentation/quartzcore/optimizing_promotion_refresh_rates_for_iphone_13_pro_and_ipad_pro)

**Comment by dreampiggy** [comment]
@lyandy @zifeizhang  Can anyone with reproducable iPhone to debug and fire PR ? Maybe iPhone simulator can not reproduce this issue.
